packages/core/src/components/DataTable/types.ts

```diff
@@ -0,0 +1,224 @@
+import React from 'react';
+import { SortDirectionType } from 'react-virtualized';
+import { WithStylesProps } from '../../composers/withStyles';
+
+export type RowHeightOptions = 'micro' | 'small' | 'regular' | 'large';
```

Comment:
this is almost a feature request, but in speaking with @elibrumbaugh I think we will have a use case for `jumbo` row sizes, would it be possible to add support for that? if you don't want to add now we can do this in a follow up PR.